survey-of-agency-tools.md

```diff
@@ -0,0 +1,149 @@
+
+Seems to be focused on creating budget reports, less about data and data
+visualizations or performance tracking.
+
+
+### IBM Cognos
+
+Used by OPM.
+
+General business intelligence [cloud-based tool](https://www.ibm.com/products/cognos-analytics).
+
+> Experience IBM Cognos Analytics, an interactive way for virtually anyone to
+> find, explore, and share data-driven insights in a governed environment. Find
+> precise and timely answers from your data or from content built by others.
+> Create compelling reports and dashboards which you can easily distribute
+> throughout your company. Use automated alerts to monitor changes to key
+> findings. Confidently and quickly take actions to improve your business.
+
+Rich features for building dashboards and custom visualizations. "Smarts" to
+guess the best visualization for your data. No mention of export or API
+features.
+
+
+### BFS
+
+Used by EPA.
+
+Private, homegrown, [internal-only tool](https://bfssb.epa.gov/Account/Login?ReturnUrl=%2f).
+
+> The Budget Formulation System (BFS) is replacing EPA’s legacy Budget
+> Automation System (BAS) system. It enables agency staff to formulate budgets,
+> integrate budgets with strategic and annual performance plans, track financial
+> actuals against established budgets, model future payroll needs, and more.
+
+Can create pre-defined visualizations and reports. Some kind of
+limited export is supported.
+
+
+### FACTSInfo
+
+[Foreign Assistance Coordination and Tracking System (FACTSInfo)](https://www.usaid.gov/data/dataset/adfec9e6-635b-435a-a0a5-1e2d2a40450a).
+
+> Foreign assistance planning and reporting system for USAID and Department of
+> State (DOS); supports both reporting needs and transactional budget planning
+> and formulation. Primary users are DOS, Office of the Global AIDS Coordinator
+> (OGAC), and USAID.
+
+Owned by the State Department and shared with USAID.
+
+Internal, private [system](http://pdf.usaid.gov/pdf_docs/PCAAB918.pdf)
+containing budget data, mission strategic plans, and operational plans.
+
+
+### Oracle BI
+
+Used by DHS.
+
+[Oracle Business Intelligence
+12c](https://www.oracle.com/solutions/business-analytics/business-intelligence/index.html).
+Focused on general business intelligence and analytics.
+
+> Foster a data-driven culture with powerful, visually stunning analytics.
+> Explore new insights and empower people across the organization to make
+> faster, more informed business decisions.
+
+Export of data and reports is not featured, but [support
+articles](https://docs.oracle.com/cloud/otbi11116/bifgen/BIFGN/exporting_report.htm)
+show that exporting individual reports and maybe raw data is possible.
+
+
+### Program Performance Tracking System
+
+Used by HHS.
+
+An online [web
+portal](https://hhs-ppts.od.nih.gov/plexus/control?fromPage=x&toPage=/jsp/applications/visualscorecard/login.jsp&actionName=Cm_GotoPage)
+for managing HHS performance, budget, and Moyer Reports.
```

Comment:
Googling "Moyer Reports" gives me a PBS show. 🤔

Reply:
Oh strange, it was supposed to give you a Rick Astley video... ;D

Reply:
I thought maybe I had misspelled it, but that is correct. It is quite difficult to find anything about it on the internet. In fact, the most helpful explanation I could find was in the PPTS documentation itself:
> The Moyer Report is a module that collect cross-cutting budgetary information so that
Congress can track how funding is being allocated across OpDivs and StaffDivs for a
particular health care topic (e.g., Diabetes)

It is HHS specific and it shows up in some random health studies. Not sure if it is produced annually or if it is made public.